test/unit/app/buy-eth-url.spec.js

```diff
@@ -17,10 +17,10 @@ describe('buy-eth-url', function () {
     network: '42',
   }
 
-  it('returns wyre url with address for network 1', function () {
+  it('returns coinbase url with amount and address for network 1', function () {
```

Comment:
Nit: This test description was fixed along with the Wyre V2 change. That should probably be preserved.